playbooks/roles/sysctl/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- block:
+    - name: Harden the kernel
```

Comment:
Maybe this should say “Get the combined sysctl values”, if I’m reading this right?

Reply:
That sounds reasonable, I can push a change.

Reply:
Even better, removed entirely.